test/Generators.hs

```diff
@@ -0,0 +1,12 @@
+{-# LANGUAGE TypeFamilies #-}
+module Generators
```

Comment:
I figure we’re gonna have a lot of generators, many of which may depend on each other.